### DISCUSSION TITLE
Aarch64 support

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [oneapi-src/oneMKL#565] [Q&A] [answered]
Good day.

Is my understanding correct that oneMKL currently does not support aarch64 architecture?

**Comment by andrii0lomakin** [comment]
I am asking because I see that you support AdapativeCpp, so I am curious if my assumption is wrong.

**Comment by s-Nick** [comment, marked as answer]
Hi @andrii0lomakin,
Thank you for your interest in oneMKL. Right now, it doesn't support aarch64 targets. The main issue I see would be maintain support and the lacking of CI for such backend.  
Given the open source nature of the project, we will be glad to have your contribution. To give you more information, ARM is part of the UXL Foundation and a aarch64 backend maybe eventually be added. It would be even more possible if it becomes an high requested feature from community and possibly a community driven effort.

**Comment by adegomme** [comment]
FYI, a pull request adding aarch64 support (for Arm performance libraries but also for netlib backend) was submitted for the BLAS domain today (lapack will follow if the first one is accepted). It was tested using intel/llvm sycl compiler and adaptivecpp.
https://github.com/uxlfoundation/oneMath/pull/629
